recording/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3"
+
+services:
+
+  nextcloud-talk-recording:
+    build:
+      context: ..
+      dockerfile: ./docker-compose/Dockerfile
+    shm_size: '2gb'
+    restart: on-failure
+    ports:
+      - "127.0.0.1:8000:8000"
+    volumes:
+      - ./files/etc/service/nextcloud-talk-recording/run:/etc/service/nextcloud-talk-recoding/run
+      - ./files/etc/nextcloud-talk-recording/server.conf:/etc/nextcloud-talk-recording/server.conf
```

Comment:
Rather than providing a copy of `server.conf` with just the `listener` configuration set it would be better to copy `server.conf.in` to `/etc/nextcloud-talk-recording/server.conf` in the Dockerfile and replace the `listen` property with something like `sed "s/#listen =.*/listen = 0.0.0.0:8000/`.

This way it would not be needed to keep `server.conf.in` and `docker-compose/files/etc/nextcloud-talk-recording/server.conf` in sync, and the Dockerfile will provide a configuration file even if the image is built without docker compose.